### PR TITLE
Rename mapping cache length metric

### PR DESCRIPTION
**Pull request by matthiasr**
The mapper package can be used outside the statsd exporter, so it is
better to use generic names here.

follow-up to #280

### CODE REVIEW

**Comment by brian-brazil** [comment]
> The mapper package can be used outside the statsd exporter, so it is better to use generic names here.

Whether it can be used elsewhere is irrelevant to metric naming. For example even if you use the Prometheus tsdb elsewhere, the correct metric name prefix is still `prometheus_tsdb_`

**Comment by matthiasr** [comment]
then let me rephrase that. this package is a generic mapper from one metric model into another. it is not _exporter_ specific.

**Comment by matthiasr** [comment]
therefore, all metrics are now under the `mapping_cache` prefix, wherever they are used.

**Comment by brian-brazil** [comment]
That's a very generic name, there should probably be a statsd or prometheus prefix in there somewhere.
